app/src/main/java/org/mozilla/fenix/experiments/Experiments.kt

```diff
@@ -4,14 +4,20 @@
 
 package org.mozilla.fenix.experiments
 
-class Experiments {
-    companion object {
-        const val A_A_NIMBUS_VALIDATION = "fenix-nimbus-validation-v3"
-        const val ANDROID_KEYSTORE = "fenix-android-keystore"
-        const val DEFAULT_BROWSER = "fenix-default-browser"
-    }
+/**
+ * Enums to identify features in the app. These will likely grow and shrink depending
+ * on the experiments we want to perform.
+ */
+enum class FeatureId(val jsonName: String) {
```

Comment:
nit: `jsonName` maybe featureName? Can we add a `@property` to describe that this is :)